templates/contribute/index.md

```diff
@@ -111,15 +113,17 @@ Here are some other frequently-used labels:
 - A **"RFC"** (= request for comment) is a PR about a change that might be controversial or need a decision from an expert about
 whether to proceed at all.
 
-- You can add **"awaiting-CI"**, which will temporarily hide the PR on the main review queue.
-  This label will be automatically removed when CI is complete.
+- You can add **"awaiting-CI"** if you're not certain whether CI will succeed.
+  This will temporarily hide the PR on the main review queue.
```

Comment:
This seems to be the first place where the "review queue" is mentioned. It might be good to explain this further, as well as mentioning / linking to the `#queue` itself (maybe near the top of this section).

Reply:
Thanks, I've updated.